docs/commands.md

```diff
@@ -0,0 +1,86 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Initialize a collection
+`notary init example.com/image`
+
+`notary publish example.com/image`
+
+## Add delegations
+`notary delegation add example.com/image targets/qa_team --all-paths <key>`
+
+Note: This will be staged as a changelist, you will need to publish for it to take effect
+
+## Remove delegations
+`notary delegation remove example.com/image targets/qa_team`
+
+Note: This will be staged as a changelist, you will need to publish for it to take effect
+
+## View status of a collection
+`notary status example.com/image`
+
+## Publish a changelist
+`notary publish example.com/image`
+
+You could also include a `-p` flag to notary commands that stage changes as changelists to auto-publish them
+
+example: `notary init -p example.com/image`
+
+## Export private key
+`notary key export --key <keyID> -o my_key.pem`
+
+## Import delegation private key
+`notary key import my_key.pem --role delegations`
+
+## Add delegation public key
+`notary delegation add example.com/image targets/qa_team --all-paths <key>`
+
+Note: This will be staged as a changelist, you will need to publish for it to take effect
+
+## Remove delegation public key
+`notary delegation remove example.com/image targets/qa_team`
+
+Note: This will be staged as a changelist, you will need to publish for it to take effect
+
+## Add a target 
+`docker push example.com/image`
+or using only notary CLI
+
+`notary add example.com/image v1:0 file.txt`
+
+`notary publish example.com/image`
+
+## Remove a target
+`notary remove example.com/image v1:0`
+
+`notary publish example.com/image`
+
+## Rotate a key (root, target, snapshot, timestamp)
+`notary key rotate example.com/image targets`
+
+## Rotate a delegation key
+### Admin actions
+`notary delegation add example.com/image targets/qa_team <newkey>`
+
+`notary delegation remove example.com/image targets/qa_team <oldkey>`
+
+`notary publish example.com/image`
+### Owner of old key after admin actions (Not always necessary depending on state)
+`notary witness example.com/image targets/qa_team`
+
+Witness marks roles to be re-signed the next time they're published. It recovers a role that has been made invalid by all keys being removed. It's only necessary if, for example, key4 was the current signer of targets/qa_team and its removal made the role's targets file invalid
```

Comment:
This comment makes no sense. I still maintain that `witness` shouldn't be described or thought of as a way to recover from key revocations.

If that is a secondary use case for `witness`, that's fine. But it can't be the primary, documented use for the command.